js/map.js

```diff
@@ -52,15 +52,27 @@ function randomPassableTile(){
 
 function generateMonsters(){
     monsters = [];
-    let numMonsters = level+1;
+    const numMonsters = level > 5 ? level+1 : 6;
```

Comment:
okay, if level is 1-4, use level+1, else use 6. This will keep levels from scaling out of control